packages/core/src/builtin-tasks/contract.ts

```diff
@@ -16,6 +16,18 @@ import { task, subtask } from '../config/config-env'
 import { paramTypes } from '../params'
 import { getUserConfigPath } from '../project-structure'
 import { getGenesisScriptsConfig } from '../util/chain'
+import { Hash } from '@ckb-lumos/base'
+
+type ContractMigration = {
+  cell_recipes: Array<{
```

Comment:
What does `recipe` mean in `migration`

Reply:
This migrations file is generated by `capsule`, and I'm not quite sure what `cell_recipes` means